From: Bernd Zeimetz <bernd@bzed.de>
Date: Sun, 24 Apr 2011 08:16:18 +0200
Subject: Remove libssl requirement from configure.

Openssl is not allowed to be linked due to the GPL2.
As it is only required for curl, we use gnutls instead.
---
 configure.in |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/configure.in b/configure.in
index af707a8..81421be 100644
--- a/configure.in
+++ b/configure.in
@@ -83,7 +83,7 @@ AC_CHECK_LIB(tiff, TIFFOpen, tiff_ok=yes, tiff_ok=no)
 fi
 AC_SUBST(LIBTIFF)
 
-pkg_modules="gtk+-2.0 >= 2.8.0 libxml-2.0 >= 2.4 gconf-2.0 >= 2.0 lcms dbus-1 exiv2 fftw3f lensfun libcurl flickcurl sqlite3 libssl libgphoto2 x11"
+pkg_modules="gtk+-2.0 >= 2.8.0 libxml-2.0 >= 2.4 gconf-2.0 >= 2.0 lcms dbus-1 exiv2 fftw3f lensfun libcurl flickcurl sqlite3 libgphoto2 x11"
 PKG_CHECK_MODULES(PACKAGE, [$pkg_modules])
 AC_SUBST(PACKAGE_CFLAGS)
 AC_SUBST(PACKAGE_LIBS)
-- 
